src/activity/exe-unit/exe-unit.ts

```diff
@@ -392,16 +390,15 @@ export class ExeUnit {
     await sleep(100, true);
 
     // Send script.
-    const results = await this.executor.execute(
+    const results = this.executor.execute(
```

Comment:
Style: check the places where you run `executor.execute` and make sure you use the `$` (like `results$`) for consistency and readability.